interfaces/Portalicious/src/main.ts

```diff
@@ -1,5 +1,10 @@
 /// <reference types="@angular/localize" />
 
+import '@angular/common/locales/global/ar';
+import '@angular/common/locales/global/en';
+import '@angular/common/locales/global/es';
+import '@angular/common/locales/global/fr';
+import '@angular/common/locales/global/nl';
```

Comment:
```suggestion
```

I don't think this is necessary. I believe it happens automagically as part of the build process. If it doesn't, let's investigate separately.